examples/plugins/app.pl

```diff
@@ -2,12 +2,15 @@
 
 use v6.c;
 use lib 'lib';
+use lib 'examples/plugins/lib';
 use Bailador;
-
-app.config.plugins = {param => 1};
+use Bailador::Plugin::Example;
+use Data::Dump;
 
 get '/' => sub {
-    template 'index.tt', { version => $version };
+    app.plugins.add('Example', Bailador::Plugin::Example.new(config => {param => 'Test'}) );
```

Comment:
this line would add the plugin for each request HTTP GET request on /. Wouldn't you only want to add this once?

Reply:
You're right that example isn't the best way to do it.
I will rewrite the sample.

Thanks for watching